src/components/themes.js

```diff
@@ -42,6 +42,15 @@ const defaultTheme = {
   button: {
     borderRadius: 3,
   },
+  avatar: {
+    borderRadius: 2,
+    borderWidth: 1,
+    size: {
+      lg: 90,
+      md: 60,
+      sm: 40,
+    },
+  },
```

Comment:
Is it necessary to save the avatar's properties in the theme? I think the theme plays the role of config. So we should hold global properties which use in more than only one place. for Avatar we can hold its properties in its own component.